src/lib/coerce.js

```diff
@@ -215,14 +215,14 @@ exports.valObjectMeta = {
         requiredOpts: ['flags'],
         otherOpts: ['dflt', 'extras', 'arrayOk'],
         coerceFunction: function(v, propOut, dflt, opts) {
-            if(typeof v !== 'string') {
-                propOut.set(dflt);
-                return;
-            }
             if((opts.extras || []).indexOf(v) !== -1) {
                 propOut.set(v);
                 return;
             }
+            if(typeof v !== 'string') {
+                propOut.set(dflt);
+                return;
+            }
```

Comment:
Could you please elaborate why this change is needed?

Reply:
Just reversing the order of short-circuits to support non-string extras https://github.com/plotly/plotly.js/pull/6193#discussion_r879534123

Reply:
As @alexcjohnson wrote:
```
OK that's pretty good evidence that we don't currently have any flaglists with non-string extras 😉
We could allow this in coerce pretty easily by pushing the extras test up above that non-string short-circuit.
```

Without this, the flaglist coerce function does not pass non-string values.